### PR TITLE
New Django Model Cheat Sheet

### PR DESCRIPTION
**Pull request by pjhampton**
### Description
The essential fields types and relationships of the data being stored when working with the Django Object Relational Mapper (ORM).

### Data Source
https://docs.djangoproject.com/en/1.9/topics/db/models/

### Example Queries
django models cheat sheet

### Screenshots 

screen shot 2016-03-18 at 21 02 59
screen shot 2016-03-18 at 21 04 02

### References

https://duck.co/ia/view/django_model_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by pjhampton** [comment]
@GuiltyDolphin is `terminal` the best template type for this IA?

**Comment by GuiltyDolphin** [comment]
@pjhampton Looks fine to me. I'm not sure if any of the others would really improve the layout. You could try `code` and `reference` locally, but I think generally `terminal` is used for this kind of thing (not sure why `terminal` happens to fit when it should probably be one of the others...).

File: share/goodie/cheat_sheets/json/django-model.json [hunk 1]
```diff
@@ -0,0 +1,137 @@
+{
+    "id": "django_model_cheat_sheet",
+    "name": "Django Model Cheat Sheet",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/django-model.json]: @pjhampton Lets remove "Cheat Sheet" from name

File: share/goodie/cheat_sheets/json/django-model.json [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        "Field Types",
+        "Field Types Cont.",
+        "Relationship Types"
+    ],
+
+    "sections": {
+        "Field Types": [
+            {
+                "key": "AutoField",
+                "val": "An IntegerField that automatically increments according to available IDs."
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/django-model.json]: @pjhampton  Avoid periods at end of val fields and description

Comment (pjhampton) [inline share/goodie/cheat_sheets/json/django-model.json]: :+1:

**Comment by moollaza** [comment]
@pjhampton Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@pjhampton this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**